### PR TITLE
Use the wikimedia mathjax clone

### PR DESCRIPTION
**Pull request by physikerwelt**
To allow for customizations a customized MathJax version is used.

### CODE REVIEW

**Comment by d00rman** [comment]
I'm guessing this should be a follow-up of [wikimedia/MathJax/#4](https://github.com/wikimedia/MathJax/pull/4) ?

**Comment by physikerwelt** [comment]
Yes. Please check if the syntax is the standard and correct.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "jsdom": "3.1.1",
         "speech-rule-engine": "*",
         "yargs": "*",
-        "MathJax": "https://github.com/mathjax/MathJax/tarball/mathjax-node-2.5"
+        "MathJax": "git+http://git@github.com/wikimedia/MathJax#mathoid-0-2-8"
```
Comment (d00rman) [inline package.json]: should be `git+https://github.com/wikimedia/MathJax#mathoid-0-2-8`

**Comment by physikerwelt** [comment]
@d00rman thanks... Should be fixed now

**Pull request merged**